### PR TITLE
Feature campaign show

### PR DESCRIPTION
**Pull request by kamalhosny**

### CODE REVIEW

File: src/app/campaigns/campaign.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+angular.module('crowdFundApp').controller('campaignController', function($scope, $http, $stateParams,$sce) {
+  $http.get('http://localhost:3000/campaign/'+$stateParams.id+'.json').then(
```
Comment (wazery) [inline src/app/campaigns/campaign.js]: Please add a space before and after the `+` sign!

File: src/app/campaigns/campaign.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    function(success){
+      $scope.campaign=success.data
```
Comment (wazery) [inline src/app/campaigns/campaign.js]: 1. Please add a space before and after the `=`
2. Please remove all the debugging lines (`console.log`)

File: src/app/campaigns/campaign.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      console.log($scope.campaign);
+      ////////////
+      // $scope.campaign.video="https://youtube.com/aqAUmgE3WyM"
+      // $scope.campaign.achieved=3;
+      ///////////////
+      if($scope.campaign.video){
+        $scope.youTubeVideoUrl=$scope.campaign.video.replace('.com/watch?v=',".com/embed/");
```
Comment (wazery) [inline src/app/campaigns/campaign.js]: 👍

**Review by wazery — CHANGES_REQUESTED** [review]
Please check the comments on this PR so I can merge it!

**Comment by kamalhosny** [comment]
@wazery  please ya 2ostaz wazery .. could u merge us in the development branch :D

**Pull request merged**